docs/source/developers/bug_reports.rst

```diff
@@ -203,59 +190,24 @@ Identify affected version
 +++++++++++++++++++++++++
 
 If you're reporting something that used to work in a previous version
-but doesn't work in the current release, you can add the **Affects version**
-field to identify the earliest known version where the bug is observed.
-For feature requests and other proposals, leave **Affects version** empty as
-it is not applicable.
+but doesn't work in the current release, you add the **Affects version**
+in the description to identify the earliest known version where the bug is
+observed.
```

Comment:
I'm not sure it makes sense to keep this paragraph. People will generally mention that it's a regression in order to get attention.